[Hyperdrive] Adjustement to pricing and limits for tier update (#21353)
cloudflare/cloudflare-docs: src/content/partials/workers/hyperdrive_pricing.mdx

---
{}
---

> import { Markdown } from "~/components";

Reviewer: ```suggestion
import { Markdown, Details } from "~/components";
```

Hyperdrive is included in both the Free and Paid [Workers plans](/workers/platform/pricing/).

> |                              | Free plan<sup>1</sup> | Paid plan |

Reviewer: ```suggestion
|                              | Free plan[^1] | Paid plan |
```

| ---------------------------- | --------------------- | --------- |

> | Database queries<sup>2</sup> | 100,000 / day         | Unlimited |

Reviewer: ```suggestion
| Database queries[^2] | 100,000 / day         | Unlimited |
```

> <sup>1</sup> The Workers Free plan includes limited Hyperdrive usage. All limits
> reset daily at 00:00 UTC. If you exceed any one of these limits, further
> operations of that type will fail with an error.
> 
> <sup>2</sup> Database queries refers to any database statement made via
> Hyperdrive, whether a query (`SELECT`), a modification (`INSERT`,`UPDATE`, or
> `DELETE`) or a schema change (`CREATE`, `ALTER`, `DROP`).

Reviewer: ```suggestion
[^1]: The Workers Free plan includes limited Hyperdrive usage. All limits reset daily at 00:00 UTC. If you exceed any one of these limits, further operations of that type will fail with an error.

[^2]: Database queries refers to any database statement made via Hyperdrive, whether a query (`SELECT`), a modification (`INSERT`,`UPDATE`, or `DELETE`) or a schema change (`CREATE`, `ALTER`, `DROP`).

<Details header="Footnotes" open={true}>
1: The Workers Free plan includes limited Hyperdrive usage. All limits reset daily at 00:00 UTC. If you exceed any one of these limits, further operations of that type will fail with an error.

2: Database queries refers to any database statement made via Hyperdrive, whether a query (`SELECT`), a modification (`INSERT`,`UPDATE`, or `DELETE`) or a schema change (`CREATE`, `ALTER`, `DROP`).
</Details>
```
